dGame/dBehaviors/BasicAttackBehavior.cpp

```diff
@@ -72,6 +81,8 @@ void BasicAttackBehavior::DoHandleBehavior(BehaviorContext* context, RakNet::Bit
 	}
 
 	if (isImmune) {
+		LOG("Target targetEntity %llu is immune! (Is this method even used?)",branch.target); //Immune is succesfully proc'd
+		//Seriously, is this ever used? I never see it in the console.
```

Comment:
even if it isnt used, as per the reverse engineered client code, this is the correct logic if this path is hit